app/src/main/java/io/intrepid/contest/screens/splash/SplashContract.java

```diff
@@ -1,18 +1,16 @@
-package io.intrepid.contest.splash;
+package io.intrepid.contest.screens.splash;
 
 import io.intrepid.contest.base.BaseContract;
 
-public interface SplashContract {
+public class SplashContract {
```

Comment:
honestly I wasn't sure. Does the outer class need to be an interface or a class? What are the implications of labeling it as an interface if the Contract itself just specifies other interfaces?